config/config.go

```diff
@@ -52,6 +52,9 @@ type Config struct {
 	TokenLimit      uint   `toml:"token-limit" json:"token-limit"`
 	OOMAction       string `toml:"oom-action" json:"oom-action"`
 	EnableStreaming bool   `toml:"enable-streaming" json:"enable-streaming"`
+	// Set sys variable lower-case-table-names, ref: https://dev.mysql.com/doc/refman/5.7/en/identifier-case-sensitivity.html.
+	// TODO: we now just ignore the system variable lower_case_table_names, we treat the table name as case sensitive actually.
```

Comment:
We actually only support mode 2, which keeps the original case, but the comparison is case-insensitive.